app/views/shared/_header.html.erb

```diff
@@ -58,12 +58,12 @@
   <%# </div> %>
   <nav class="usa-nav grid-col-fill flex-fill nav-container">
       <ul class="usa-nav__primary usa-accordion flex-justify-end">
-          <li class="usa-nav__primary-item margin-right-305 hidden">
+          <li class="usa-nav__primary-item margin-right-305">
```

Comment:
i _**think**_ with Go Fish being gone now, we probably just want to convert this to a link to Shark Tank, rather than having "Bulletin" dropdown.
Probably check with Alex and Becca.

Reply:
I updated it to just a link for "Shark Tank".

![image](https://user-images.githubusercontent.com/60527788/81755943-1f8f7f00-946f-11ea-8587-e649285f2517.png)